bike-erp/backend/src/main/dao/BikeDao.ts

```diff
@@ -0,0 +1,36 @@
+import db from "../helpers/db";
+
+export class BikeDao {
+    public createBike = (
+        price: number,
+        size: string,
+        color: string,
+        description: string,
+        grade: string,
+        quantity: number
+    ) => {
+        return new Promise<any>((resolve, rejects) => {
+            const insert =
+                "INSERT INTO `bike` (`price`, `size`, `color`, `bike_description`, `grade`, `quantity`) VALUES ('" +
+                price +
+                "', '" +
+                size +
+                "', '" +
+                color +
+                "', '" +
+                description +
+                "', '" +
+                grade +
+                "', '" +
+                quantity +
+                "');";
+            db.query(insert, (err) => {
+                if (err) {
+                    rejects(err);
+                } else {
+                    resolve({ message: "Bike inserted succesfully." });
```

Comment:
can you also add the `rows` attribute into your resolve. This will facilitate handling the account receivable since I need the ID. Like this `response: JSON.parse(JSON.stringify(rows))`

Reply:
response with id was added

Reply:
Done